### PR TITLE
added algorithm visualiser app 

### LINKED ISSUE #244: Algorithm visualiser app
## 🚀 Feature

A algorithm visualiser app which visually shows and hence helps in better understanding the logic being sorting algorithms.

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

## Motivation

I always wanted to be able to see how the algorithms perform base on their logic.

## Pitch

Understanding the logic behind a sorting algo and implementing it is good but for beginners having a visual way helps grasp the concept much better and faster.

### PR DESCRIPTION
**Pull request by Phoenix-031**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes

### Description

Added sorting algorithm visualiser app with consisting of various sorting algorithms

Fixes #244

### Checklist

- [ ✅] I've read the contribution guidelines.
- [ ✅] I've checked the issue list before deciding what to submit.
- [ ✅] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

None

### CODE REVIEW

**Review by photon149 — APPROVED** [review]
@Phoenix-031 Interactive Visuals 👍🏻 
@prakhar3062 please have a look , approved from my end.

**Review by prakhar3062 — COMMENTED** [review]
@Phoenix-031 nice work . okay from my side

**Comment by Phoenix-031** [comment]
@prakhar3062 thankyou for your approval do i have to do anything else or will it get merged in due time?

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Phoenix-031 Good project , One thing when i ran this project it shows only `bubble sort` and screen got  freezed. Can You mention in your `readme.md` file that how user can use or play with the output . 
And reduce the image size whose are in `readme.md` file .

**Comment by Phoenix-031** [comment]
Sure I am adding the process of using this to the readme file and i will be resizing the gifs as much as possible.
It shows bubble sort first bcuz that is set to be the default algo u can change that by pressing M or S or I or Q etc as mentioned at the top of the screen .
I also ran the program again i couldn't find anything that would freeze the program.

File: Desktop Application/Intermediate/Python/Algorithm_visualiser/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+![](https://img.shields.io/badge/Programming_Language-Python-blue.svg)
+![](https://img.shields.io/badge/Main_Tool_Used-Pygame-gold.svg)
+![](https://img.shields.io/badge/Python_Version-3.10-brown.svg)
+
+
+# <h2 align = "center">Hello Programmer's</h2><br><br>
+
+# <p align = "center">I made a Algorithm Visualiser with the help of pygame module in python, contains mostly all the famous sorting algorithms.</p>
+
+![alt-text](./Images/AlgorithmVisualiser.png)
```
Comment (Kushal997-das) [inline Desktop Application/Intermediate/Python/Algorithm_visualiser/README.md:10]: This link is not working !

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Change !!

**Comment by Phoenix-031** [comment]
added the image now it'll be visible :)

**Review by Kushal997-das — APPROVED** [review]
@Phoenix-031 💯

**Pull request merged**